todo_app/data/sample_calls

```diff
@@ -1,459 +0,0 @@
-https://trello.com/b/NLvhLBOS/todoapp
-
-https://developer.atlassian.com/cloud/trello/rest/api-group-actions/
-https://developer.atlassian.com/cloud/trello/guides/rest-api/api-introduction/
-
-GetAllBoards(verbose)
-https://api.trello.com/1/members/me/boards?key=b4b0f437afe756ad8944b7aedfbe3cf4&token=946719d7da9b126dd37539a72e97f92c1298f73cbb70a0eb365c7f4fdd732829
```

Comment:
Was this a real key and token? If so, we shouldn't have missed this before, and we should definitely make sure we rotate your Trello credentials now!

Reply:
Done!